src/pip/_internal/cli/cmdoptions.py

```diff
@@ -75,8 +75,9 @@ def getname(n):
         return getattr(check_options, n, None)
 
     names = ["build_options", "global_options", "install_options"]
-    if any(map(getname, names)):
-        control = options.format_control
+
+    control = getattr(options, "format_control", None)
```

Comment:
Why is `format_control` not present on the `options` object? Is there a way we can make it always available instead?

Reply:
I am unsure, as this approach is the one being used with the `build/global/install_options`